Felienne/hedy
# Fix: Class name can't be empty or duplicate

Provide a general summary of your changes in the Title of the PR

**Description**
In this PR we add two additional checks to the changing of a class name to improve user experience. First, we verify that the name can't be empty. This was possible, resulting in a class we were not able to click as it didn't have a name. Next we verify if the chosen class name isn't already existing for this teacher. Duplicate class names by the same teacher are undesirable and shouldn't be allowed.

**Fix for**
This PR fixed #1152 and fixed #1278 

**How to test**
Sign in with a teacher account and on the teachers page select one of your classes. Press the button to rename the class and leave it either empty of type the name of one of your other classes. In the first scenario a modal with the `class_name_empty` message should be returned. In the second scenario a modal with the `class_name_duplicate` should appear. Notice that the second message will also appear if the chosen name is identical to the previous name.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
